src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -1983,10 +1983,10 @@ class NodeUtils(object):
             else:
                 total = size_as_int(hpmemtotal)
         except Exception:
-            pbs.logmsg(pbs.EVENT_DEBUG,
+            pbs.logmsg(pbs.EVENT_DEBUG3,
```

Comment:
Default log level includes DEBUG2, but not DEBUG3. I think the admin should be aware that huge page calculation failed without having to alter the $logevent setting.

Reply:
@mike0042 :  but on a system without Hugepagesize set, on default log level, this message would keep appearing, and there's really no way to turn it off. I was being consistent with what we do in another case:
if not resc_vntype and not file_vntype:
            pbs.logmsg(pbs.EVENT_DEBUG3,
                       '%s: Could not determine vntype' % caller_name())

Reply:
Fair enough. If someone wants it changed in the future, they can file an issue.